src/Database/Expression/QueryExpression.php

```diff
@@ -376,6 +376,7 @@ public function in($field, $values, $type = null)
      * @param array $types associative array of types to be associated with the values
      * passed in $values
      * @return $this
+     * @deprecated 3.6.0 use \Cake\Database\Query::newCaseExpr()
```

Comment:
Should this also emit a deprecation warning? Or are you planning for the method to last until 5.0?

Reply:
Forgot about the warning, I'll add it in a commit